helix-view/src/decorations.rs

```diff
@@ -0,0 +1,29 @@
+use crate::graphics::Style;
+
+#[derive(Clone, Copy, PartialEq)]
+pub enum TextAnnotationKind {
+    /// Add to end of line
+    Eol,
+    /// Replace actual text or arbitary cells with annotations.
+    /// Specifies an offset from the 0th column.
+    Overlay(usize),
```

Comment:
Is this inline annotation?

Reply:
No, it overwrites normal text with virtual text, which would be useful for #274 to show jump labels.